tools/Dockerfile

```diff
@@ -137,7 +69,26 @@ RUN set -uex && \
   apt-get install nodejs -y && \
   npm install -g ajv-cli@v3.3.0 json-dereference-cli@0.1.2 zx
 
+# Copy binaries and scripts
+COPY tools/download-binaries.sh .
```

Comment:
Add comment

# single script ensures one layer. Note that docker-squash does not support ARM arch.

Reply:
perhaps this script is not needed because the last docker build stage copy from previous stages.